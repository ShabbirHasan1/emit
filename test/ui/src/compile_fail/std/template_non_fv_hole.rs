fn main() {
    emit::emit!("template {true}");
}
