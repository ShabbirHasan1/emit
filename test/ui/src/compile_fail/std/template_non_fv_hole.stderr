error: parsing failed: failed to parse `true` as a field-value expression
 --> src/compile_fail/std/template_non_fv_hole.rs:2:28
  |
2 |     emit::emit!("template {true}");
  |                            ^^^^
